src/taxonomy/data/thunks.js

```diff
@@ -0,0 +1,15 @@
+// @ts-check
+import { getTaxonomyExportFile } from './api';
+
+/**
+ * Downloads the file of the exported taxonomy
+ * @param {number} pk
+ * @param {string} format
+ * @returns {void}
+ */
+/* istanbul ignore next */
+const exportTaxonomy = (pk, format) => (
+  getTaxonomyExportFile(pk, format)
+);
```

Comment:
@ChrisChV IIRR, `thunk.js` is for redux operations ([ref](https://redux.js.org/usage/writing-logic-thunks)). And since `exportTaxonomy` is just wrapping an API function, can't we just call `getTaxonomyExportFile` directly where it's needed?

Is that right @xitij2000 ?

Reply:
Yes, thunks are where you can connect the API to the redux store. Since this is using a different approach with useQuery, I think there might not be any thunks here.

Reply:
Thanks! Ok, I agree. I deleted `thunks.js` https://github.com/openedx/frontend-app-course-authoring/pull/645/commits/080b03ec36a5d019f475bf31a9a5fe5791973e9b